src/App.js

```diff
@@ -1,25 +1,22 @@
 import React from 'react';
-import logo from './logo.svg';
+import {Row, Col, Layout} from 'antd';
+import BotonUpload from './Components/upload/'
+import Buscador from './Components/search/'
```

Comment:
Si bien el código funciona sin estos, acordate de siempre poner los ; al final
```suggestion
import BotonUpload from './Components/upload/';
import Buscador from './Components/search/';
```